apps/browser-extension-wallet/src/stores/slices/stake-pool-search-slice.ts

```diff
@@ -40,7 +40,11 @@ const fetchStakePools =
         startAt: skip,
         limit: limit - skip + 1
       },
-      sort
+      // @ts-expect-error TODO remove when ticker sort is available
+      sort: sort && {
+        ...sort,
+        ...(sort.field === 'ticker' ? { field: 'name' } : {})
```

Comment:
```suggestion
        ...(sort.field === SortField.ticker ? { field: SortField.name } : {})
```
if these are available of course*

Reply:
Thank you. That's a valid comment, but we no longer have `enum` for this. We know how to transform `enum` to union-type but the other way around seems tricky or impossible (we haven't investigated this in-depth) and we prefer to reuse/extend the union from the SDK (for safety).

Reply:
let's wait till our discussion in clack is resolved